packages/cli-dxns/src/dxos.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package dxos;
+
+message DXNS_cid {
+  bytes data = 1;
+}
+
+message IPFS_cid {
+  bytes data = 1;
+}
+
+message Any {
+  DXNS_cid schema_cid = 1;
+  string mesage_fqn = 2;
+  bytes payload = 3;
+}
+
+message RecordAttributes {
+  string name = 1;
+  string description = 2;
+  string version = 3;
+  string author = 4;
+}
+
+message KUBE {
+  RecordAttributes attributes = 1;
+  string url = 2;
+
+  Any extension = 10;
+}
+
+message Service {
+  RecordAttributes attributes = 1;
+  string type = 2;
+
+  Any extension = 10;
+}
+
+message App {
+  RecordAttributes attributes = 1;
+  string build = 2;
+  IPFS_cid binary_hash = 3;
```

Comment:
Name the field `ipfs_cid` as well? Binary hash is not really descriptive of its purpose

Reply:
@Marik-D I think binary here refers to the asset (e.g., JS bin). perhaps deployment?

```
bytes deployment_cid
```

Or (including comments above)

```
message Deployment {
  string platform = 1;
  string plaform_version = 2;
  bytes binary_cid = 3;
}
```